boards.txt

```diff
@@ -20649,3 +20649,210 @@ esp32c3m1IKit.menu.EraseFlash.all=Enabled
+Edgebox-ESP-100.build.defines=
+Edgebox-ESP-100.build.loop_core=
+Edgebox-ESP-100.build.event_core=
+Edgebox-ESP-100.build.psram_type=qspi
+Edgebox-ESP-100.build.memory_type={build.boot}_{build.psram_type}
+
+Edgebox-ESP-100.menu.PSRAM.disabled=Disabled
+Edgebox-ESP-100.menu.PSRAM.disabled.build.defines=
+Edgebox-ESP-100.menu.PSRAM.disabled.build.psram_type=qspi
+Edgebox-ESP-100.menu.PSRAM.enabled=QSPI PSRAM
+Edgebox-ESP-100.menu.PSRAM.enabled.build.defines=-DBOARD_HAS_PSRAM
+Edgebox-ESP-100.menu.PSRAM.enabled.build.psram_type=qspi
+Edgebox-ESP-100.menu.PSRAM.opi=OPI PSRAM
+Edgebox-ESP-100.menu.PSRAM.opi.build.defines=-DBOARD_HAS_PSRAM
+Edgebox-ESP-100.menu.PSRAM.opi.build.psram_type=opi
+
+Edgebox-ESP-100.menu.FlashMode.qio=QIO 80MHz
+Edgebox-ESP-100.menu.FlashMode.qio.build.flash_mode=dio
+Edgebox-ESP-100.menu.FlashMode.qio.build.boot=qio
+Edgebox-ESP-100.menu.FlashMode.qio.build.boot_freq=80m
+Edgebox-ESP-100.menu.FlashMode.qio.build.flash_freq=80m
+Edgebox-ESP-100.menu.FlashMode.qio120=QIO 120MHz
+Edgebox-ESP-100.menu.FlashMode.qio120.build.flash_mode=dio
+Edgebox-ESP-100.menu.FlashMode.qio120.build.boot=qio
+Edgebox-ESP-100.menu.FlashMode.qio120.build.boot_freq=120m
+Edgebox-ESP-100.menu.FlashMode.qio120.build.flash_freq=80m
+Edgebox-ESP-100.menu.FlashMode.dio=DIO 80MHz
+Edgebox-ESP-100.menu.FlashMode.dio.build.flash_mode=dio
+Edgebox-ESP-100.menu.FlashMode.dio.build.boot=dio
+Edgebox-ESP-100.menu.FlashMode.dio.build.boot_freq=80m
+Edgebox-ESP-100.menu.FlashMode.dio.build.flash_freq=80m
+Edgebox-ESP-100.menu.FlashMode.opi=OPI 80MHz
+Edgebox-ESP-100.menu.FlashMode.opi.build.flash_mode=dout
+Edgebox-ESP-100.menu.FlashMode.opi.build.boot=opi
+Edgebox-ESP-100.menu.FlashMode.opi.build.boot_freq=80m
+Edgebox-ESP-100.menu.FlashMode.opi.build.flash_freq=80m
+
+Edgebox-ESP-100.menu.FlashSize.4M=4MB (32Mb)
+Edgebox-ESP-100.menu.FlashSize.4M.build.flash_size=4MB
+Edgebox-ESP-100.menu.FlashSize.8M=8MB (64Mb)
+Edgebox-ESP-100.menu.FlashSize.8M.build.flash_size=8MB
+Edgebox-ESP-100.menu.FlashSize.8M.build.partitions=default_8MB
+Edgebox-ESP-100.menu.FlashSize.16M=16MB (128Mb)
+Edgebox-ESP-100.menu.FlashSize.16M.build.flash_size=16MB
+#Edgebox-ESP-100.menu.FlashSize.32M=32MB (256Mb)
+#Edgebox-ESP-100.menu.FlashSize.32M.build.flash_size=32MB
+
+Edgebox-ESP-100.menu.LoopCore.1=Core 1
+Edgebox-ESP-100.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+Edgebox-ESP-100.menu.LoopCore.0=Core 0
+Edgebox-ESP-100.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+Edgebox-ESP-100.menu.EventsCore.1=Core 1
+Edgebox-ESP-100.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+Edgebox-ESP-100.menu.EventsCore.0=Core 0
+Edgebox-ESP-100.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+Edgebox-ESP-100.menu.USBMode.default=Hardware CDC and JTAG
+Edgebox-ESP-100.menu.USBMode.default.build.usb_mode=0
+Edgebox-ESP-100.menu.USBMode.hwcdc=USB-OTG (TinyUSB)
+Edgebox-ESP-100.menu.USBMode.hwcdc.build.usb_mode=1
```

Comment:
Can you change it back to the default values as is for ESP32-S3 dev module?
So please switch 0/1 and change `Edgebox-ESP-100.build.usb_mode=0` to 1.
It is better to keep these settings the same. This won't have any impact on your board definition.

Reply:
The latest commit should have the changes as request, Thanks.